### DISCUSSION TITLE
v2.6.3 and FontAwesome

### DISCUSSION THREAD

**Discussion by cloudguy23** [templaza/astroid-framework#292] [General]
Hello.

With respect to Astroid Font Awesome compatibility - is this something thats natively included part of the framework and zero template, or does one have to add FA manually as per instructions outlined in:
https://github.com/templaza/astroid-framework/discussions/186#discussion-4359968

I'm using v2.6.3 and zero template.  My objective is to use FontAwesome in articles / content on my site, asking to clarify scope of FontAwesome if it's for site-wide use or only within template configuration.  

Thank you.

**Comment by Chacapamac** [comment]
I @cloudguy23 
Astroid Awesome font integration are control by the **plugin** —> **Astroid Plugin**

**There you can choose how to load your Awesome fonts.**
![Screen Shot 2023-02-14 at 11 21 16 AM](https://user-images.githubusercontent.com/1907276/218796499-f63df5fd-c51e-4073-9231-4622c959d346.jpg)

• Load from the web by using “**CDN**”
• Load the locally Astroid Awesome library “**Local**” 
•“**Off**” Turn the Astroid Awesome font system off

I personally use “**Local**” because I want to have the Awesome fonts loading visually in my **JCE Editor** edition window for a better WYSIWYG experience for the website administrators and editors.
I simply add this type of call in my **editor custom css** file

e.g. _@import "../../../../../media/astroid/assets/vendor/fontawesome/css/all.min.css";_

You can turn all that off and load your own local library by using the “**OFF**” setting in the plugin;
• Download Font Awesome last version —> https://fontawesome.com/download 
• or any other version —>  https://github.com/FortAwesome/Font-Awesome/releases 
• Copy the entire /webfonts folder and the /css/all.css into your “media” Yourtemplate font folder

![Screen Shot 2023-02-14 at 11 51 23 AM](https://user-images.githubusercontent.com/1907276/218803570-b4b2a124-2146-4755-9dd5-f6e43aef7f6c.jpg)
• Add a reference to the copied /css/all.css file into the <head> of each template or page that you want to use Font Awesome on by using your Astroid Template Manager custom css using the “**Before </head>** ” section. and add this type of call.

![Screen Shot 2023-02-14 at 11 56 54 AM](https://user-images.githubusercontent.com/1907276/218805304-9d7d37e3-e7c7-4786-8cf6-621f0df0d566.jpg)

**Comment by Chacapamac** [comment]
@cloudguy23  Do you think we can mark this discussion as “SOLVED” ?

**Comment by cloudguy23** [comment]
Thank you for this clarification.   Issue may be marked as solved.

**Reply by Chacapamac** [reply]
@cloudguy23  Thank You!
**Good Day!**
